handler/src/test/java/io/netty/handler/ssl/JdkSslEngineTest.java

```diff
@@ -263,6 +267,40 @@ public String select(List<String> protocols) {
         }
     }
 
+    @Test
+    public void testEnablingAnAlreadyDisabledSslProtocol() throws Exception {
```

Comment:
I'd like to, but the assert statements are different for JDK/OpenSSL. Any suggestions? Would I do an ```instanceof``` check on the ```sslEngine``` variable?